request_test.go

```diff
@@ -137,3 +144,110 @@ func TestRequestUnmarshalJSON(t *testing.T) {
 		assert.Equal(t, tc.expectedError, err, tc.scenario)
 	}
 }
+
+func TestSimplePOSTItem(t *testing.T) {
```

Comment:
Those 2 tests are great but seem more related to the `Collection` struct than the `Request` struct (as you're basically creating a collection, filling it and writting it). Maybe we could move them into the `collection_test.go` file.

Reply:
done.